### ISSUE TITLE
Code Coverage edge case - Multiline use statments

### ISSUE THREAD

**Issue by Garbee** [sebastianbergmann/phpunit-documentation#422]
I think I've found a new edge-case for the [code coverage](https://phpunit.de/manual/current/en/code-coverage-analysis.html) that should get documented in case others hit it.

<details>
<summary>Example Code</summary>

```php
<?php
use PHPUnit\Framework\TestCase;

// Because it is "line based" and not statement base coverage
// the variables for "use" are shown as not covered.
$someObject->reduce(function () use (
    $outerItem,
    $anotherOuterItem
) {
   return false;
});

// You either need to reduce it to a single-line statement
// Or add @codeCoverageIgnore[Start|End] comments
//@codeCoverageIgnoreStart
$someObject->reduce(function () use (
    $outerItem,
    $anotherOuterItem
) {
    //@codeCoverageIgnoreEnd
    return false;
});
?>
```

</details>

If there is a way to fix this without compressing into a single-line I'd love to hear it. Otherwise I feel this warrants documenting since it does seem quirky.

Thank you,
Garbee

**Comment by sebastianbergmann** [comment]
@derickr What do you think?

**Comment by derickr** [comment]
I think: https://bugs.xdebug.org/view.php?id=1403

**Comment by sebastianbergmann** [comment]
Thanks, @derickr.

**Comment by derickr** [comment]
And fixed for Xdebug 2.5.2: https://github.com/xdebug/xdebug/pull/332

**Comment by Garbee** [comment]
Oh fantastic!

Thank you so much, I didn't look into filing an issue upstream. Figured it would have been a known issue. 

Have a great day.
